docs/ru/engines/database-engines/materialized-postgresql.md

```diff
@@ -82,3 +86,10 @@ ENGINE = MaterializedPostgreSQL('postgres1:5432', 'postgres_database', 'postgres
 
 SELECT * FROM postgresql_db.postgres_table;
 ```
+## Примечания {#notes}
+
+### Сбой слота логической репликации {logical-replication-slot-failover}
+
+Слоты логической репликации, которые есть на основном сервере, не доступны на резервных репликах.
+Поэтому в случае сбоя новый основной сервер (который раньше был резевным физическим сервером) не будет знать о слотах, которые были расположены на вышедшем из строя основном сервере. Это приведет к нарушению репликации из PostgreSQL.
```

Comment:
```suggestion
Поэтому в случае сбоя новый основной сервер (который раньше был резевным) не будет знать о слотах репликации, которые были созданы на вышедшем из строя основном сервере. Это приведет к нарушению репликации из PostgreSQL.
```

Reply:
also may be add this example https://github.com/ClickHouse/ClickHouse/issues/28529#issuecomment-927209693 (with addition of  the author nickname)